.travis.yml

```diff
@@ -3,5 +3,15 @@ dist: trusty
 language: node_js
 node_js:
   - "6.9.5"
+install:
+  # node packages
+  - npm install
+  # dotnet
+  - sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list' 
+  - sudo apt-key adv --keyserver apt-mo.trafficmanager.net --recv-keys 417A0893
+  - sudo apt-get update
+  - sudo apt-get install dotnet-dev-1.0.0-rc4-004769 -y
+  # gulp
+  - npm install gulp
```

Comment:
I had some trouble with gulp yesterday hence the 2 installs, will try removing this.